error.sourceName must not be null

I'm trying to use the plugin, but I'm getting an error. 

My code in _build.gradle_ is:
```
plugins {
    id 'com.android.application'
    id "org.gradlewebtools.minify" version "1.2.1"
}
...
task jsMinify(type: org.gradlewebtools.minify.JsMinifyTask) {
    srcDir = project.file("$projectDir/src/main/assets/Token/app")
    dstDir = project.file("$projectDir/src/main/assets/Token/appmin")
    options.originalFileNames = true
    options.compilationLevel = com.google.javascript.jscomp.CompilationLevel.SIMPLE_OPTIMIZATIONS
}
```


The error I'm getting is:

```
Execution failed for task ':app:jsMinify'.
> error.sourceName must not be null

* Try:
Run with --info or --debug option to get more log output. Run with --scan to get full insights.

* Exception is:
org.gradle.api.tasks.TaskExecutionException: Execution failed for task ':app:jsMinify'.
Caused by: java.lang.NullPointerException: error.sourceName must not be null
	at org.gradlewebtools.minify.minifier.result.Event.<init>(Event.kt:24)
	at org.gradlewebtools.minify.minifier.result.Error.<init>(Error.kt:8)
	at org.gradlewebtools.minify.minifier.js.JsMinifier.minifyFile(JsMinifier.kt:40)
	at org.gradlewebtools.minify.minifier.Minifier.minifyFileInternal(Minifier.kt:76)
	at org.gradlewebtools.minify.minifier.Minifier.handleFile(Minifier.kt:69)
	at org.gradlewebtools.minify.minifier.Minifier.minifyInternal(Minifier.kt:40)
	at org.gradlewebtools.minify.minifier.Minifier.minify(Minifier.kt:28)
	at org.gradlewebtools.minify.JsMinifyTask.minify(JsMinifyTask.kt:25)
```

Anyone could help me to bypass the problem?

--- a/src/main/kotlin/org/gradlewebtools/minify/minifier/result/Event.kt
+++ b/src/main/kotlin/org/gradlewebtools/minify/minifier/result/Event.kt
@@ -21,10 +21,10 @@ open class Event {
         charNo = error.charno
         lineNo = error.lineNumber
         message = error.description
-        sourceFileName = error.sourceName
+        sourceFileName = error.sourceName ?: ""
     }
 
     override fun toString(): String {
-        return "$message at $sourceFileName line $lineNo : $charNo"
+        return "$message at $sourceFileName line $lineNo:$charNo"
     }
 }

